services/core/cards-publication/src/main/modeling/swagger.yaml

```
@@ -336,6 +336,16 @@ definitions:
         parameters:
           EN: Summary of card content
           FR: Resume du contenu de la carte
+  CardCreationReport:
+    type: object
+    description: Created card identifiers
+    properties:
+      id:
+        type: string
+        description: Id of the card
```

```suggestion
        description: ID of the process instance to which this card refers
```
Feel free to change the wording if you have a better idea, I just think we need to make more apparent what the difference between id & uid is.